src/gui/qgsexpressionbuilderwidget.h

```diff
@@ -287,8 +219,9 @@ class GUI_EXPORT QgsExpressionBuilderWidget : public QWidget, private Ui::QgsExp
      * Returns a pointer to the dialog's function item model.
      * This method is exposed for testing purposes only - it should not be used to modify the model.
      * \since QGIS 3.0
+     * \deprecated since QGIS 3.14
      */
-    QStandardItemModel *model();
+    Q_DECL_DEPRECATED QStandardItemModel *model();
```

Comment:
Don't forget SIP_DEPRECATED too. AKA the "Trinity of Deprecation".

Reply:
done